From ba8bf33534c09fd33d991dcaaf1e2174c432a7bf Mon Sep 17 00:00:00 2001
From: Rong Bao <rong.bao@csmantle.top>
Date: Wed, 11 Feb 2026 15:06:17 +0800
Subject: [PATCH 02/70] Arch Linux kernel v6.19.2-arch0

---
 Makefile | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/Makefile b/Makefile
index f0eb659930b2c..91099d4e0efd7 100644
--- a/Makefile
+++ b/Makefile
@@ -2,7 +2,7 @@
 VERSION = 6
 PATCHLEVEL = 19
 SUBLEVEL = 2
-EXTRAVERSION =
+EXTRAVERSION = -arch0
 NAME = Baby Opossum Posse
 
 # *DOCUMENTATION*
-- 
2.53.0

