From b3b1e9bb120a0cd72a8d1e51f7655ce1d16bfb1f Mon Sep 17 00:00:00 2001
From: Rong Bao <rong.bao@csmantle.top>
Date: Wed, 11 Feb 2026 15:06:17 +0800
Subject: [PATCH 02/70] Arch Linux kernel v6.19.3-arch0

---
 Makefile | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/Makefile b/Makefile
index 21df635071198..59f1e2b7adcd9 100644
--- a/Makefile
+++ b/Makefile
@@ -2,7 +2,7 @@
 VERSION = 6
 PATCHLEVEL = 19
 SUBLEVEL = 3
-EXTRAVERSION =
+EXTRAVERSION = -arch0
 NAME = Baby Opossum Posse
 
 # *DOCUMENTATION*
-- 
2.53.0

